app/graphql/types/search_result/global_scope/target_type.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+Types::SearchResult::GlobalScope::TargetType = GraphQL::UnionType.define do
```

Comment:
What about this?
GlobalSearchEntry -> RankedGlobalSearchEntry
GlobalSearchTarget -> GlobalSearchEntry